### ISSUE TITLE
Data analysis/simple Machine Learning tutorial

### ISSUE THREAD

**Issue by maciejkula** [rust-community/rustbridge#24]
A short tutorial that walks through reading a small dataset from disk, then training and testing some simple machine learning models.

Spam detection would be a classic example.

**Comment by skade** [comment]
Assume _no_ previous knowledge.

But, given that, simple classification sounds like something that can be done. (sorting email into buckets for example).

**Comment by maciejkula** [comment]
Yes, I'll assume no Rust knowledge (but I guess assuming some rudimentary ML knowledge is fine in the same way a web service tutorial would assume knowledge about requests, responses, maybe templating?).

There is a nice little spam/non-spam dataset for SMS messages which should be easy to use.

**Comment by erickt** [comment]
@maciejkula did a first pass for this in https://github.com/rust-community/rustbridge/tree/master/workshops/machine_learning.
